apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/util/AuthUserPasswordCheckerTest.java

```diff
@@ -30,7 +30,7 @@
 
   @Before
   public void setup() {
-    checker = new AuthUserPasswordChecker();
+    checker = new AuthUserPasswordChecker(null);
```

Comment:
I think we need to pass a mock portal config here and test when it has `listOfCodeFragment`